extension/typedefs/shopify/CollectionRepositoryCommandGet.d.ts

```diff
@@ -1,6 +1,9 @@
+import {ShopifyProductImagesImage} from "shopify-api-node-interface/shopify-api-node-interface";
```

Comment:
Can be removed because its unused (?)

Reply:
And please check your codestyle for typescript files: File => Settings => Editor => "Code Style" => Typescript and reformat your changes

Reply:
done